src/utils/Api.js

```diff
@@ -0,0 +1,126 @@
+class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getAppInfo() {
+    //Call getUserInfo in this array
+    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
+  }
+
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+    }
+    Promise.reject(`Error: ${res.status}`);
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    }).then(this._checkResponse);
+  }
+
+  getUserInfo() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      headers: this._headers,
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        Promise.reject(`Error: ${res.status}`);
```

Comment:
Please, use `.then(this._checkResponse);` in every request, to avoid the same `ok` checking